include/deal.II/base/quadrature.h

```diff
@@ -137,7 +137,7 @@ class Quadrature : public Subscriptor
    * Move constructor. Construct a new quadrature object by transferring the
    * internal data of another quadrature object.
    */
-  Quadrature (Quadrature<dim> &&) = default;
+  Quadrature (Quadrature<dim> &&) = default; // NOLINT
```

Comment:
Hm, these places are annoying indeed. The compiler should be able to figure out whether a `=default` constructor is `noexcept` or not simply because it knows which functions exactly it calls. I would consider this a compiler bug.

But I guess we have other places where we already work around compiler bugs in a variety of ways...